install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,23 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes should have a label signifying the router. The registry should be
+installed on them. For example:
+
+====
+----
+# default selectors for router and registry services
+# openshift_router_selector='region=infra'
+# openshift_registry_selector='region=infra'
+----
+====
+
+The default router and registry are automatically deployed. The default label is
+`*region=infra*`. In all other aspects, this is a normal node. Not all nodes are
```

Comment:
The router / registry will only be automatically deploy if Nodes exist with labels that match the selector settings above.  Here's an example of how ansible labels can apply labels to Nodes: https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.ose.example#L289